ci-operator/step-registry/cucushift/installer/check/vsphere/etcd-static-pods/cucushift-installer-check-vsphere-etcd-static-pods-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+check_etcd_static_pods() {
+    local static_pods_num=0
+    set -x
+    master_node_name=$(oc get nodes | grep master | awk -F' ' '{print $1}')
+    etcd_pods_name=$(oc get pod -n openshift-etcd | grep 'Running' | awk -F' ' '{print $1}')
+    for name in $master_node_name
+    do
+       num=$(echo $etcd_pods_name | grep $name | wc -l)
+       static_pods_num=$((static_pods_num + num))
+    done
+    set +x
+    #The expected number is specified by env MASTER_REPLICAS
+    if [ X"$static_pods_num" != X"${MASTER_REPLICAS}" ]; then
```

Comment:
Hmm, I think we can drop the static etcd pods check point from installer test case any more, I think that check points should be already covered by etcd team.

Reply:
ok. so, for this case, just add the job without any check right?

Reply:
I think so, if the case is to cover 5 master cases, the answer is "yes", but actually I do not have idea why we cover 5 masters case, especially using UPI method. cc @jinyunma to confirm if have more ideas.

Reply:
ok. i see

Reply:
This case is added from 4.1, and I could not open [jira issue](https://jira.coreos.com/browse/PROD-789) linked in the case, but from the case step and explanation of parameter 'controlPlane.replicas' in [official installation doc](https://docs.openshift.com/container-platform/4.15/installing/installing_vsphere/upi/installing-vsphere.html#installation-vsphere-config-yaml_installing-vsphere), the main checkpoint for this case is to make sure that the number of ectd endpoints matches with control plane nodes number.

>The number of control plane machines that you add to the cluster. Because the cluster uses this values as the number of etcd endpoints in the cluster, the value must match the number of control plane machines that you deploy.

Reply:
Seems that ControlPlane.replicas is not updated in this PR, still 3 (hardcoded)
https://github.com/openshift/release/pull/49913/files#diff-2c64592135d669befc354cc0f50c2752c663695bda3345afb0f06d079060062eL226.

The installation with 5 master nodes deployed still succeeded and pass the post-check of etcd endpoint @WenXinWei added before, right?

Reply:
> Seems that ControlPlane.replicas is not updated in this PR, still 3 (hardcoded) https://github.com/openshift/release/pull/49913/files#diff-2c64592135d669befc354cc0f50c2752c663695bda3345afb0f06d079060062eL226.
> 
> The installation with 5 master nodes deployed still succeeded and pass the post-check of etcd endpoint @WenXinWei added before, right?

yes.

Reply:
From [Installation configuration parameters for vsphere](https://docs.openshift.com/container-platform/4.15/installing/installing_vsphere/installation-config-parameters-vsphere.html#installation-configuration-parameters-optional_installation-config-parameters-vsphere), supuport value for `ControlPlane.replicas` is 3 or 1 when deploying single-node ocp.

So I guess maybe we don't need to run this case, @jianlinliu @gpei wdyt?

Reply:
Agree to disable the test case. 

But from this PR, I guess we can still work on it to get UPI/IPI customized `MASTER_REPLICAS` and `WORKER_REPLICAS` to be supported.

Reply:
> we can still work on it to get UPI/IPI customized MASTER_REPLICAS and WORKER_REPLICAS to be supported.

+1, current update is only for UPI part. For IPI, need more change in https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

Reply:
yes, and the changes to remove control-plane machinesets is also required for 4.16 UPI jobs now.

Reply:
got it. will update accordingly. thanks.

Reply:
also suggest to rename env as below to keep consistent with other platforms.
`MASTER_REPLICAS` -> `CONTROL_PLANE_REPLICAS`
`WORKER_REPLICAS` -> `COMPUTE_NODE_REPLICAS`